gui/tools/Helper.h

```diff
@@ -0,0 +1,227 @@
+#include <algorithm>
+#include <vector>
+#include <sstream>
+#include <string>
+
+// VTK includes
+#include <vtksys/SystemTools.hxx>
+#include "vtkMatrix3x3.h"
+#include "vtkMatrix4x4.h"
+#include "vtkNew.h"
+#include "vtkTransform.h"
+
+namespace help
+{
+  //----------------------------------------------------------------------------
+  // Description
+  // Split string by separated char
+  static void SplitString(const std::string &s, char delim,
```

Comment:
It seems like we should have this in kwsys (vital), or elsewhere, but I haven't checked.